doc/source/serve/doc_code/deployment_graph_control_flow_based_on_user_inputs.py

```diff
@@ -0,0 +1,36 @@
+import ray
+from ray import serve
+from ray.experimental.dag.input_node import InputNode
+
+
+ray.init()
+serve.start()
+
+
+@serve.deployment
+class Model:
+    def __init__(self, weight):
+        self.weight = weight
+
+    def forward(self, input):
+        return input + self.weight
+
+
+@serve.deployment
+def combine(value_refs, combine_type):
+    values = [ray.get(value_ref) for value_ref in value_refs]
+    if combine_type == "sum":
+        return sum(values)
+    else:
+        return max(values)
+
+
+with InputNode() as user_input:
+    model1 = Model.bind(0)
+    model2 = Model.bind(1)
+    output1 = model1.forward.bind(user_input[0])
+    output2 = model2.forward.bind(user_input[0])
+    dag = combine.bind([output1, output2], user_input[1])
+
+print(ray.get(dag.execute(1, "max")))
```

Comment:
we're using default access by index for InputNode that we haven't documented in cookbook or anywhere else yet. I suggest we either change this to a list, or add inline comments / explanation of the text that by default we convert multiple values of `dag.execute()`  to a data object that can be accessed by index or key.